docs/tools/mbed_cli_2/migration_guide.md

```diff
@@ -0,0 +1,188 @@
+
+<span style="background-color:#ffcccc;">The subcommand `mbed-cli toolchain`
+(set or get default toolchain) is deprecated. Pass the desired toolchain on the
+command line when compiling with `mbed-tools compile --toolchain GCC_ARM`, to,
+for example, select `GCC_ARM` as the toolchain.</span><br>
+
+<span style="background-color:#ffcccc;">The subcommand `mbed-cli target` (set
+or get default target)" is deprecated. Pass the desired target on the command
+line when compiling with `mbed-tools compile --mbed-target
+DISCO_L475VG_IOT01A`, to, for example, select `DISCO_L475VG_IOT01A` as the
+Mbed target.</span><br>
+
+
+### Creating an Mbed program
+<span style="background-color:#ffe6cc;">The subcommand `mbed-tools new` creates
+a new Mbed program by default and supports only one command-line-option
+`--create-only` (create a program without fetching mbed-os). The other
+command-line-options supported in Mbed CLI 1, as follow, are deprecated:
+
+* `--scm` (now always git)
+* `--program` (now always a program)
+* `--library` (now always a program)
+* `--mbedlib` (instead, add with a separate subsequent command)
+* `--depth` (no longer needed, as the Mbed CLI 2 automatically uses the shallowest clone or
+fetch possible)
+* `--protocol` (determined from the URL instead)
+* `--offline` (not needed, as there is no longer any Mbed-specific git cache)
+* `--no-requirements` (Mbed CLI 2 never automatically installs requirements)
+</span><br>
+
+### Importing an Mbed program
+<span style="background-color:#ffe6cc;">The subcommand `mbed-tools
+import` clones an Mbed project and library dependencies. The following
+command-line-options supported in Mbed CLI 1 are deprecated:
+
+* `--depth`: The new CLI optimistically imports the most shallow repository
+  possible. Running `git clone --depth <depth> <repo_url> | cd <repo_name> &&
+  mbed-tools deploy` will create an mbed project with the required depth.
+* `--protocol`: To select the protocol, import using the URL for the
+  desired protocol. For instance, the tool allows you to import projects or
+  libraries over SSH as follows `mbed-tools import
+  git://github.com/ARMmbed/<some_driver>`. Alternatively, if you would prefer
+  to use SSH you can add the following into your .gitconfig to force the tool
+  to use it instead.
+
+```sh
+[url "ssh://git@github.com/"]
+  insteadOf = https://github.com/
+```
+
+* `--offline`: The new tool doesn't maintain a git cache so we don't have the
+  option to use locally cached repositories.
+* `--no-requirements`
+* `--insecure`
+
+</span><br>
+
+### Configuring an Mbed program
+
+<span style="background-color:#ccffcc;">The subcommand `mbed-tools configure`
+generates an Mbed OS config CMake file named `mbed_config.cmake` in the path
+`cmake_build/<TARGET_NAME>/\<PROFILE>/\<TOOLCHAIN>/`.</span><br>
+
+### Compiling an Mbed program
+
+<span style="background-color:#ffcccc;">The subcommand "mbed-tools
```

Comment:
Should be ``` `mbed-tools compile` ``` for consistency, there are a couple of examples of this in this section which use"" instead.